packages/styles/src/link/index.css

```diff
@@ -0,0 +1,82 @@
+/**
+ * Link styles
+ * @variant {small | medium | large} syn-link The size of the link
+ * @boolean { false } syn-link--quiet Applies the quiet link styling
+ * @boolean { false } syn-link--disabled Applies the disabled link styling
+ */
+.syn-link {
+  align-items: center;
+  color: var(--syn-link-color);
+  display: inline-flex;
+  font: var(--syn-body-medium-regular);
+  gap: var(--syn-spacing-x-small);
+}
+
+/* Use inline text styling when there is no syn-icon */
+.syn-link:not(:has(syn-icon)) {
+  display: inline;
+}
+
+.syn-link:not(.syn-link--disabled):hover {
+  color: var(--syn-link-color-hover);
+}
+
+.syn-link:not(.syn-link--disabled):active {
+  color: var(--syn-link-color-active);
+}
+
+.syn-link:not(.syn-link--disabled):focus-visible {
+  outline: var(--syn-focus-ring);
+  outline-offset: var(--syn-focus-ring-offset);
+}
+
+/* Overrides for smaller links */
+.syn-link--small {
+  font: var(--syn-body-small-regular);
+  gap: var(--syn-spacing-2x-small);
+}
+
+/*
+ * Overrides for medium links.
+ * Note: The default styles are already set for medium links, just here for completeness
+ */
+.syn-link--medium {
+  font: var(--syn-body-medium-regular);
+  gap: var(--syn-spacing-x-small);
+}
+
+.syn-link--large {
+  font: var(--syn-body-large-regular);
+  gap: var(--syn-spacing-small);
+}
+
+/* Quite Variant of links use other colors */
+.syn-link--quiet {
+  color: var(--syn-link-low-emphasis-color);
+}
+
+.syn-link--quiet:hover {
+  color: var(--syn-link-low-emphasis-color-hover);
+}
+
+.syn-link--quiet:active {
+  color: var(--syn-link-low-emphasis-color-active);
+}
+
+/* Styling for disabled links */
+.syn-link--disabled {
+  cursor: not-allowed;
+  opacity: var(--syn-input-disabled-opacity);
+}
+
+/*
+ * We set pointer-events to none to prevent the link from being clicked
+ * This is done in :active to allow using a custom cursor
+ */
+.syn-link--disabled:active {
+  pointer-events: none;
+}
+
+.syn-link--disabled:focus {
+  outline: none;
+}
```

Comment:
```suggestion
/* Styling for disabled links */
.syn-link--disabled:not([href]), a.syn-link:not([href]) {
  cursor: not-allowed;
  opacity: var(--syn-input-disabled-opacity);
}
```

Your suggestion would lead to inaccessible HTML, as the element would still be tabbable.

I suggest to make this only werk if no `href` is set and in addition to make it more convenient that dropping `href` on a link makes it disabled out of the box.

Reply:
Sounds like a good idea!